spec/openapi.yaml

```diff
@@ -7711,6 +7711,309 @@ paths:
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CustomFields'
+              examples:
+                response:
+                  value:
+                    response:
+                      code: SUCCESS
+                      data:
+                        items:
+                          - fieldUid: ngahq6dgh4ed
+                            type: SELECTBOX
+                            fieldName: field-name
+                            enabled: true
+                            required: true
+                            searchable: true
+                            displayToTranslators: true
+                            options:
+                              - option1
+                              - option2
+                            defaultValue: default-field-name
+                            description: custom field description
+                        totalCount: 1
+          description: OK
+        400:
+          $ref: '#/components/responses/Error400ResponseDefinition'
+        401:
+          $ref: '#/components/responses/Error401ResponseDefinition'
+        429:
+          $ref: '#/components/responses/Error429ResponseDefinition'
+        500:
+          $ref: '#/components/responses/Error500ResponseDefinition'
+
+  /jobs-api/v3/accounts/{accountUid}/custom-fields/{fieldUid}:
+    put:
+      summary: Update custom field in account
+      description: >-
+        Updates custom field in account using field UID specified
+      tags:
+        - Jobs
+      operationId: updateCustomField
+
+      parameters:
+        - description: A unique identifier for an account.
+          in: path
+          name: accountUid
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateCustomField'
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CustomFields'
+              examples:
+                response:
+                  value:
+                    response:
+                      code: SUCCESS
+                      data:
+                        items:
+                          - fieldUid: ngahq6dgh4ed
+                            type: SELECTBOX
+                            fieldName: field-name
+                            enabled: true
+                            required: true
+                            searchable: true
+                            displayToTranslators: true
+                            options:
+                              - option1
+                              - option2
+                            defaultValue: default-field-name
+                            description: custom field description
```

Comment:
period please

Reply:
i think we don't need a period here because it's an example of a 'description' field value